stub_uploader/metadata.py

```diff
@@ -283,23 +284,37 @@ def verify_external_req(
     upstream_distribution: Optional[str],
     _unsafe_ignore_allowlist: bool = False,  # used for tests
```

Comment:
You didn't touch this code, but I think the same as my comment below could apply:
```suggestion
    *,
    _unsafe_ignore_allowlist: bool = False,  # used for tests
```